syntax = "proto3";
package gr;

message Notify_Connection_Id
{
	int32 connection_id = 1;
}

message Player_Login
{
	int32 char_id = 1;
	int32 ipv4 = 2;
}

message Inter_Server_Register
{
	int32 server_type = 1;
}

message Notify_Obj_Master_Player_Login
{
	int32 char_id = 1;
}

message Gatemgr_Apply_Fastest_Gate
{
	int32 client_connection_id = 1;
}

message Server_Info
{
	string ip = 1;
	int32 port = 2;
}